__tests__/fixtures/env-vars/underreact.config.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = () => ({});
```

Comment:
Would it be ok for an Underreact app *not* to have a `underreact.config.js` file if this is all it would do?

Reply:
This was fixed https://github.com/mapbox/underreact/pull/22/files#diff-cd723c9fac3aa166c5875d4ce93b4fceR13 🍬